README.md

```diff
@@ -1,56 +1,13 @@
-# Swate-templates
+# Swate Templates
 
-A collection of minimal information standard templates for Swate.
+A collection of minimal information standard templates for [Swate](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/implementation/Swate.html).
 
 The templates in this repository are crawled by [Swobup](https://github.com/nfdi4plants/Swobup) and written into the production Swate database.
 
-Anyone can access them via the [template search](https://nfdi4plants.github.io/Swate-docs/docs/UserDocs/Docs05-Templates.html) function.
+Anyone can access them via the [template search](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/SwateManual/Docs05-Templates.html) function.
 
```

Comment:
Maybe add the new "releases" mechanic here for direct json access.

Reply:
Sorry, I don't know yet what that is.